java/src/main/java/com/google/cloud/dataproc/templates/kafka/README.md

```diff
@@ -237,4 +237,82 @@ bin/start.sh \
+
+## 4. Kafka To BQ via spark Direct stream
+
+General Execution:
+
+```
+GCP_PROJECT=<gcp-project-id> \
+REGION=<region>  \
+SUBNET=<subnet>   \
+GCS_STAGING_LOCATION=<gcs-staging-bucket-folder> \
+HISTORY_SERVER_CLUSTER=<history-server> \
+bin/start.sh \
+-- --template KafkaToBQDstream \
+--templateProperty project.id=<gcp-project-id> \
+--templateProperty kafka.bootstrap.servers=<kafka broker list> \
+--templateProperty kafka.topic=<kafka topic name> \
+--templateProperty kafka.starting.offset=<latest | earliest> \
+--templateProperty kafka.bq.stream.output.mode=<Append | Overwrite | ErrorIfExists | Ignore> \
+--templateProperty kafka.gcs.batch.interval=<Batch interval of the stream> \
+--templateProperty kafka.gcs.consumer.group.id=<Consumer group id for the kafka topic> \
+--templateProperty kafka.bq.dataset=kafkatobq \
+--templateProperty kafka.bq.table=kafkaevents \
+--templateProperty kafka.bq.temp.gcs.bucket=<gcs-bucket-name> \
+--templateProperty kafka.bq.await.termination.timeout=<stream-await-termination-timeout>
+```
+
+### Configurable Parameters
+Following properties are avaialble in commandline or [template.properties](../../../../../../../resources/template.properties) file:
+
+```
+# Kafka to BigQuery via Dstream
+
+# Kafka servers
+kafka.bootstrap.servers=<kafka broker list>
+
+# Kafka topics
+kafka.topic=<kafka topic names>
+
+# BigQuery output dataset
+kafka.bq.dataset=<output bigquery dataset>
+
+# BigQuery output table
+kafka.bq.table=<output bigquery table>
+
+# Cloud Storage bucket name, for storing temporary files
+kafka.bq.temp.gcs.bucket=<cloud storage bucket name>
+
+# Offset to start reading from. Accepted values: "earliest", "latest" 
+kafka.bq.starting.offset=<kafka-starting-offset>
+
+# Waits for specified time in ms before termination of stream
+kafka.bq.await.termination.timeout=<stream-await-termination-timeout>
+
+# Ouptut mode for writing data. Accepted values: 'overwrite', 'append', 'ignore', 'error', 'errorifexists', 'default'
+kafka.bq.stream.output.mode=<output-mode> default value : append
+```
+
+### Example submission
+```
+export GCP_PROJECT=dp-test-project
+export REGION=us-central1
+export SUBNET=test-subnet
+export GCS_STAGING_LOCATION=gs://dp-templates-kakfatogcs/stg
+export GCS_OUTPUT_PATH=gs://dp-templates-kafkatogcs/output/
+bin/start.sh \
+-- --template KafkaToBQDstream \
+--templateProperty project.id=$GCP_PROJECT \
+--templateProperty kafka.bootstrap.servers=102.1.1.20:9092 \
+--templateProperty kafka.topic=events-topic \
+--templateProperty kafka.starting.offset=latest \
+--templateProperty kafka.bq.stream.output.mode=Append \
+--templateProperty kafka.bq.batch.interval=60000 \
+--templateProperty kafka.bq.consumer.group.id=test.group.id \
+--templateProperty kafka.bq.dataset=kafkatobq \
+--templateProperty kafka.bq.table=kafkaevents \
+--templateProperty kafka.bq.temp.gcs.bucket=<gcs-bucket-name> \
+--templateProperty kafka.bq.await.termination.timeout=<stream-await-termination-timeout>
```

Comment:
Add some dummy values in Example Submission. Replace `<gcs-bucket-name>` and `<stream-await-termination-timeout>`:
```
--templateProperty kafka.bq.temp.gcs.bucket=<gcs-bucket-name> \
--templateProperty kafka.bq.await.termination.timeout=<stream-await-termination-timeout>
```

Reply:
Done